src/AddResident.js

```diff
@@ -103,57 +110,57 @@ class AddResidentForm extends Component {
             })
             .then((data) => {
-              let villageArray = JSON.parse(data);
-              let prevVillage = -1;
-              let villages = [];
-              let villageHouses = {};
-              let houses = [];
+                let villageArray = JSON.parse(data);
+                let prevVillage = -1;
+                let villages = [];
+                let villageHouses = {};
+                let houses = [];
 
-              if (villageArray.length > 0) {
-                let village = {};
-                village.label = villageArray[0].NAME;
-                village.value = villageArray[0].VID;
-                villages.push(village);
-                prevVillage = villageArray[0].VID;
-              }
+                if (villageArray.length > 0) {
+                    let village = {};
+                    village.label = villageArray[0].NAME;
+                    village.value = villageArray[0].VID;
+                    villages.push(village);
+                    prevVillage = villageArray[0].VID;
+                }
 
-              villageArray.forEach(house => {
-                if (house.VID !== prevVillage) {
-                  villageHouses[prevVillage] = houses
-                  let houseOption = {}
-                  houseOption.label = house.HOUSE_NUM;
-                  houseOption.value = house.HOUSE_NUM;
-                  houses = [houseOption]
-                  let village = {};
-                  village.label = house.NAME;
-                  village.value = house.VID;
-                  villages.push(village);
-                  prevVillage = house.VID;
-                } else {
-                  let houseOption = {}
-                  houseOption.label = house.HOUSE_NUM;
-                  houseOption.value = house.HOUSE_NUM;
-                  houses.push(houseOption);
+                villageArray.forEach(house => {
+                    if (house.VID !== prevVillage) {
+                        villageHouses[prevVillage] = houses
+                        let houseOption = {}
+                        houseOption.label = house.HOUSE_NUM + " (" + this.handleVillageBool(house.VACANT) + ")";
+                        houseOption.value = house.HOUSE_NUM;
+                        houses = [houseOption]
+                        let village = {};
+                        village.label = house.NAME;
+                        village.value = house.VID;
+                        villages.push(village);
+                        prevVillage = house.VID;
+                    } else {
+                        let houseOption = {}
+                        houseOption.label = house.HOUSE_NUM + " (" + this.handleVillageBool(house.VACANT) + ")";
+                        houseOption.value = house.HOUSE_NUM;
+                        houses.push(houseOption);
+                    }
+                });
+                if (villageArray.length > 0) {
+                    villageHouses[prevVillage] = houses;
```

Comment:
I do not see why anything except `houseOption.label = house.HOUSE_NUM + " (" + this.handleVillageBool(house.VACANT) + ")";` is in the diff